scripts/release/test_generate_winget_manifests.rb

```
@@ -0,0 +1,85 @@
+      path = Pathname(tmpdir).join("tally_checksums.txt")
+      path.write(<<~TEXT)
+        ABCDEF tally_0.26.0_Windows_x86_64.exe
+        123456 tally_0.26.0_Windows_arm64.exe
+      TEXT
+
+      assert_equal(
+        {
+          "tally_0.26.0_Windows_x86_64.exe" => "ABCDEF",
+          "tally_0.26.0_Windows_arm64.exe" => "123456",
+        },
+        read_checksums(path),
```

RuboCop (1.85.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)